proprietary/design-tokens/src/component/utrecht/data-list.tokens.json

```diff
@@ -1,26 +1,40 @@
 {
   "utrecht": {
     "data-list": {
+      "border-color": {},
+      "border-width": {},
       "margin-block-end": {
         "value": "{utrecht.space.block.md}"
       },
       "margin-block-start": {
         "value": "{utrecht.space.block.md}"
       },
       "item-key": {
+        "border-right-color": {},
+        "border-right-width": {},
         "color": {},
         "font-size": {},
         "font-weight": {
           "value": "{utrecht.typography.weight-scale.bold.font-weight}"
         },
-        "line-height": {}
+        "line-height": {},
+        "padding-block-end": {},
+        "padding-block-start": {},
+        "padding-inline-end": {},
+        "padding-inline-start": {}
       },
       "item-value": {
         "color": {},
         "font-size": {},
-        "line-height": {}
+        "line-height": {},
+        "padding-block-end": {},
+        "padding-block-start": {},
+        "padding-inline-end": {},
+        "padding-inline-start": {}
       },
       "rows": {
+        "border-bottom-color": {},
+        "border-bottom-width": {},
```

Comment:
All these empty `{}` don't have much use here for the Utrecht theme, but you might have included them so they contribute to the automatically generated design tokens documentation.

All new tokens should be included in the `tokens.json` file next in the `components/` package, in this case here:
https://github.com/nl-design-system/utrecht/blob/main/components/data-list/tokens.json